src/ScreenSelectMaster.cpp

```diff
@@ -460,7 +460,7 @@ void ScreenSelectMaster::UpdateSelectableChoices()
 
 		for ( PlayerNumber &p : vpns)
 		{
-			if(disabled && m_iChoice[p] == c)
+			if(disabled && m_iChoice[p] == static_cast<int32_t>(c))
```

Comment:
Let's do the `numeric_limits` stuff here as well.

Reply:
Can you clarify what you're looking for here?

Reply:
Re-reading I'm not even sure >_> let's cast to int instead of int32_t though